src/engraving/dom/measure.cpp

```diff
@@ -309,6 +310,23 @@ void Measure::setStaffStemless(staff_idx_t staffIdx, bool stemless)
     }
 }
 
+AutoOnOff Measure::hideStaffIfEmpty(staff_idx_t staffIdx) const
+{
+    MStaff* staff = mstaff(staffIdx);
+
+    return staff ? staff->hideIfEmpty() : AutoOnOff::AUTO;
+}
+
+void Measure::setHideStaffIfEmpty(staff_idx_t staffIdx, AutoOnOff hideIfEmpty)
+{
+    MStaff* staff = mstaff(staffIdx);
+
+    if (staff) {
+        staff->setHideIfEmpty(hideIfEmpty);
+        triggerLayout(staffIdx);
```

Comment:
I would assert `staff`, unless there are cases where it's ok for it to be null. Also, I normally try to do as little as possible inside setters and do the `triggerLayout` in some higher-up more generic functions like `setProperty`.